tests/api/printf.cpp

```diff
@@ -34,6 +34,9 @@ static std::string stdoutFileName;
 #define BUFFER_SIZE 1024
 static char stdoutBuffer[BUFFER_SIZE];
 
+std::vector<char> GLOBAL_BUFFER;
```

Comment:
clvk does not have any global variables written in full uppercase anywhere. Please use a format already use somewhere else in the repo. I would say `globalBuffer`?